app/models/user.rb

```diff
@@ -0,0 +1,8 @@
+class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
```

Comment:
- [ ][OPTIONAL] It would be nice if a user has to confirm his email before using the app :+1:

Reply:
it would be nice tom address this issue optional changes shouldnt be ignored for they help improve your coding skills